drivers/adc-dac/ad5592r/ad5592r-base.c

```diff
@@ -3,7 +3,7 @@
  *   @brief  Implementation of AD5592R Base Driver.
```

Comment:
Could you explain in the commit message, briefly, why the `gpio_map` was removed?